WcaOnRails/app/models/competition.rb

```diff
@@ -1190,6 +1201,24 @@ def distance_adjacent_to?(c, distance_km)
     self.kilometers_to(c) < distance_km
   end
 
+  def registration_open_adjacent_to?(c, distance_minutes)
+    if !c.has_registration_start_date? || !self.has_registration_start_date?
+      return false
+    end
+    self.minutes_until_other_registration_starts(c).abs < distance_minutes
+  end
+
+  def minutes_until_other_registration_starts(c)
+    if !c.has_registration_start_date? || !self.has_registration_start_date?
+      return false
+    end
+    seconds_until = (c.registration_open - self.registration_open).to_i
+    if seconds_until < 0
+      seconds_until = (self.registration_open - c.registration_open).to_i * -1
+    end
+    seconds_until / 60
+  end
```

Comment:
This might look confusing, but it follows the same logic as `days_until_competition` above: https://github.com/thewca/worldcubeassociation.org/pull/7198/files#diff-3148368ff9102d87290f301299773c8e08ebbedccadec8ec1e93b9c1c35fa470R1174